src/buildCommon.js

```diff
@@ -151,12 +151,19 @@ const mathDefault = function(
             return makeSymbol(
                 value, fontName, mode, options,
                 classes.concat("amsrm", options.fontWeight, options.fontShape));
-        } else { // if (font === "main") {
+        } else if (font === "main" || !font) {
```

Comment:
Is the `!font` necessary?  It looks like all symbols defined in `symbol.js` have a font of either `"ams"` or `"main"`.

Reply:
It is the same line that prevented the failing test mentioned above.  
In some special cases font was undefined. It looked strange for me but I assumed this is needed in somewhere else so I did not change the old behaviour.

Reply:
My guess is that we don't define a font for a lot of the unicode characters that we all now.  @ronkok is that correct?

Reply:
> don't define a font for a lot of the unicode characters that we [allow].

@kevinbarabash  I think our Unicode work as falling into two groups. 

One group began with your PR #410 that added support for Greek characters already in KaTeX fonts. My work has followed in that path, adding support for as many Unicode characters as I could, but always characters that are in KaTeX fonts. In this string of PRs, we have always specified a font.

The other group of PRs began with your PR #508 for CJK and was extended, especially by @davidflanagan and lately by @edemaine's PRs #1217 and #1278 that allow, in non-strict mode, all Unicode code points. This string of PRs does not, to my knowledge, specify a font, but rather lets the browser pick a system font. And I believe that font metrics are assumed to match the Latin letter "M".